### PR TITLE
ci: add script to fetch git org name

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This can be used by other repos to figure out the name of the org.

### CODE REVIEW

**Review by pchandra19 — APPROVED** [review]

File: scripts/git-org-name.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    echo "$gitUrl" | awk -F/ '/git/ { print $1 }' | cut -d':' -f2
+  elif echo $gitUrl | grep -qa "^https://"; then
+    echo "$gitUrl" | awk -F/ '/https/ { print $4 }'
+  else
+    die "Unknown git url type: '$gitUrl'"
+  fi
+}
+
+orgName() {
+  local originUrl=$(git config --get remote.origin.url)
```
Comment (niladrih) [inline scripts/git-org-name.sh]: Is the remote always guaranteed to be 'origin'? Is this meant for automation only? I call the openebs org 'upstream' in my dev setup.

**Review by niladrih — APPROVED** [review]
Left one comment, but LGTM.

File: scripts/git-org-name.sh (hunk 1 above)
Comment (pchandra19) [reply scripts/git-org-name.sh]: Since we clone , I would say yes.

Comment (tiagolobocastro) [reply scripts/git-org-name.sh]: It's meant for the CI so we can name dockerhub credentials and also dockerhub repository name.
We can add a manual repository override for local image building if required.

**Pull request merged**